cloud_functions/token_refresh/streams/strava.py

```
@@ -0,0 +1,79 @@
+    Args:
+        db: Firestore client.
+        uid: Firestore user id.
+        new_tokens: New tokens.
+    """
+    user_ref = db.collection("users").document(uid)
+    user_ref.update(
+        {
+            "stream=strava.accessToken": new_tokens["accessToken"],
+            "stream=strava.refreshToken": new_tokens["refreshToken"],
+            "stream=strava.expiresAt": new_tokens["expiresAt"],
+        }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)